lpd/templates/question.html

```diff
@@ -0,0 +1,117 @@
+{% load lpd_filters %}
+{% load lpd_tags %}
+
+<div class="question" data-question-id="{{ question.id }}" data-question-type="{{ question.type }}">
+
+  <p>{{ question.question_text }}</p>
+
+  {% if question.type == 'essay' %}
+    <textarea class="answer-text">{% get_answer question learner %}</textarea>
+  {% endif %}
+
+  {% if question.type == 'short-answer' %}
+    <input type="text" class="answer-text" value="{% get_answer question learner %}"/>
+  {% endif %}
+
+  {% if question.type == 'mcq' %}
+    <div class="mc-options">
+      {% for answer_option in question.get_answer_options %}
+        <label>
+          {{ answer_option.option_text }}
+          {% get_data answer_option learner as answer_option_data %}
+          {% if answer_option.allows_custom_input %}
+            <input type="text"
+                   name="{{ question.id }}-custom-input"
+                   class="custom-input"
+                   value="{{ answer_option_data.custom_input }}" />
+          {% endif %}
+          <input type="radio"
```

Comment:
This is a matter of preference I guess, so please take this comment with a grain of salt, but I find it more intuitive when the checkbox is on the left side of the option ;)

Reply:
@tomaszgy You're right, somehow that's more appealing visually (even without any custom CSS to style the questions)! Made that change: 6d8063066b8b223101089c18b9b8c90425e671fb.